src/PlaceListCardImage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import roomJpg from "./assets/img/explore/room.jpg";
+import roomWebp from "./assets/img/explore/room.webp";
+
+interface placeListImg {
+    superHost: Boolean
+}
+
+function PlaceListCardImage(props: placeListImg) {
+    return (
+        <>
+            <div className="card__img">
+                {/* <img src="./img/explore/room.webp" type="image/webp" className="lazyload" width="100%" height="auto" alt="room" /> */}
+                <picture>
```

Comment:
this is nice!

Reply:
why?

Reply:
because it let's the browser choose the best match for the image, it's backward compatible with best image for new browsers and older technique for older ones

Reply:
yes